File: pyneat-rs/test-samples/sample.go
package main

import (
    "fmt"
    "os/exec"
    "database/sql"
)

func main() {
    // Command Injection - GO-SEC-001
    cmd := exec.Command("sh", "-c", "ls -la " + userInput)
    
    // SQL Injection - GO-SEC-002
    query := "SELECT * FROM users WHERE id = " + userId
    
    // Hardcoded Secret - GO-SEC-004
    password := "super_secret_123"
    apiKey := "AKIAIOSFODNN7EXAMPLE"
    
    // Insecure TLS - GO-SEC-006
    tlsConfig := &tls.Config{InsecureSkipVerify: true}
    
    // MD5 Hash - GO-SEC-012
    hash := md5.New()
}

File: pyneat-rs/test-samples/sample.php
// // Debug output // FIXME: removed debug output<?php
// SQL Injection test
$user_id = $_GET['id'];
$query = "SELECT * FROM users WHERE id = " . $user_id;
$result = mysqli_query($conn, $query);

// XSS test
// echo $_POST['name']; // FIXME: removed debug output
// Command injection
$file = $_GET['file'];
exec("cat " . $file);

// Hardcoded secret
// $password = "super_secret_123"; // FIXME: removed debug output
// Debug output
// echo "Debug: starting..."; // FIXME: removed debug output// var_dump($data); // FIXME: removed debug output); // FIXME: removed debug output
// Eval usage
eval($_POST['code']);

// Weak hashing
// $hash = md5($password); // FIXME: removed debug output
// Object injection
$data = unserialize($_COOKIE['user_data']);
?>
